src/main/java/eggventory/StockList.java

```diff
@@ -48,6 +48,7 @@ public void addStock(String stockType, String stockCode, int quantity, String de
 
         // "Uncategorised" is always the first element on stockList.
         stockList.get(0).addStock("Uncategorised", stockCode, quantity, description);
```

Comment:
hMMM having stocks know what type they are might be strange. should discuss later

Reply:
I feel its a bit weird to create an uncategorised stockType during runtime. But I don't have a better idea too.